ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Motivation
```

Comment:
It's not clear to me why these files are shown here in the pull-request again, they are already part of next. They are not shown in the one against master. I wonder if this is an indication that there is a problem and we are diverging the two branches again or if this is just how it is.

Reply:
Pull requests done form a clean repo after a fork. 188 files modified looks normal as I reset the doc directory, before it was more something like 1500 files. After I don't know, we should just try ;)

Reply:
It might very well be completely fine, it's just weird as the other PR shows 182 files, which is correct, as the 6 additional files here are already on next, so they should not be displayed. But it might be that github just displays this weirdly and from a git point of view it is fine.